x-pack/plugins/observability/common/utils/formatters/formatters.test.ts

```diff
@@ -5,9 +5,63 @@
  * 2.0.
  */
 
-import { asPercent, asDecimalOrInteger } from './formatters';
+import { asDecimal, asInteger, asPercent, asDecimalOrInteger } from './formatters';
 
 describe('formatters', () => {
+  describe('asDecimal', () => {
+    it.each([
+      [Infinity, 'N/A'],
+      [-Infinity, 'N/A'],
+      [null, 'N/A'],
+      [undefined, 'N/A'],
+      [NaN, 'N/A'],
+    ])(
+      'displays the not available label when the number is not finite',
+      (value, formattedValue) => {
+        expect(asDecimal(value)).toBe(formattedValue);
+      }
+    );
+
+    it.each([
+      [0, '0.0'],
+      [0.005, '0.0'],
+      [1.23, '1.2'],
+      [12.34, '12.3'],
+      [123.45, '123.5'],
+      [1234.56, '1,234.6'],
+      [1234567.89, '1,234,567.9'],
```

Comment:
What about negative numbers, should we add some test cases here?

Reply:
I'd prefer to address with a separate PR as explained here if you don't mind? https://github.com/elastic/kibana/pull/122072#discussion_r778067707

Reply:
Sorry, I misunderstood it and I thought that the comment was only for the transaction rate method.